docs/css/docs.css

```diff
@@ -120,6 +120,33 @@
   margin-bottom: 0 !important;
 }
 
+/* example modal */
+#example-modal {
+  background-color: rgba(0, 0, 0, 0.8);
```

Comment:
`css variables`  can be used

Reply:
This value is used only once in whole documentation website, hence directly used.

Reply:
Okay !